app/controllers/records_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class RecordsController < ApplicationController
+  include Records::Filters
+
+  before_action :authorize, :not_guest, :not_author, :not_owner, :not_manager
+
+  respond_to :html, :json
+
+  # GET /records
+  def index
+    @records = records.limit(request.xhr? && 10).order(created_at: :desc).page params[:page]
```

Comment:
```suggestion
    @records = records.order(created_at: :desc).page params[:page]
```
Esto solo cuando respondemos con JSON le ponemos limit, para limitar la cantidad de resultados en los autocomplete.

Reply:
perfecto. Lo tenia asi el otro y se lo deje por las dudas jajaja